src/exchange/billing/account-renew/renew.html

```diff
@@ -0,0 +1,117 @@
+<div class="px-5 pt-5" data-ui-view>
```

Comment:
Is there any reason to not having padding-bottom to this `ui-view`?

I mean `p-5` instead of `px-X + pt-X`.

Reply:
No, it was inherited from the previous page.
Updated